libs/pika/debugging/include/pika/debugging/print.hpp

```diff
@@ -19,6 +19,20 @@
 #include <type_traits>
 #include <utility>
 #include <vector>
+//
+#if defined(__linux) || defined(linux) || defined(__linux__)
+#include <sys/mman.h>
+#include <unistd.h>
+#elif defined(__APPLE__)
+#include <crt_externs.h>
+#include <unistd.h>
+#define environ (*_NSGetEnviron())
+#elif defined(PIKA_WINDOWS)
+#include <winsock.h>
+#define environ _environ
+#else
+extern char** environ;
+#endif
```

Comment:
Sorry, one last thing. This looks unused and duplicates what's in `pika/debugging/environ.hpp`. Also a leftover?
```suggestion
```

Reply:
Feel free to press "Commit suggestion" if this should be removed.